### ISSUE TITLE
[T16][Accessibility][Performance] Unable to Enter Data

### ISSUE THREAD

**Issue by ranjithswrk** [Mohanme13/Feb-19-Bug-bash#252]
Steps to Recreate:
Bot able to enter data in Gravida by Para in OBGYN Tab.
